spec/functional/resource/git_spec.rb

```diff
@@ -239,6 +239,29 @@ def expect_branch_to_be(branch)
     end
   end
 
+  context "when updating a branch already checkout out" do
+    it "checks out master, commit ot the repo and checkout the last changes" do
```

Comment:
```suggestion
    it "checks out master, commits to the repo, and checks out the latest changes" do
```

Reply:
Sorry if I'm misunderstanding, but it seems like this is describing what the test is doing rather than the behavior the test is supposed to demonstrate. Does this accurately describe the changes you made in this PR?
```suggestion
    it "syncs changes made remotely" do
```